GPRegistryPolicyParser.psm1

```diff
@@ -22,6 +22,14 @@ data LocalizedData
 }
 
 Import-LocalizedData  LocalizedData -filename GPRegistryPolicyParser.Strings.psd1
+# Add compatibility for Powershell Core which does not support 'encoding byte' via splatting
+if ( $PSVersionTable.PSVersion.Major -le "5")
+    {
+        $byteParam = @{Encoding = "Byte"}
+    } elseif ($PSVersionTable.PSVersion.Major -gt "5")
+    {
+        $byteParam = @{AsByteStream = $True}
+}
 
```

Comment:
Used the same version detection logic merged in earlier commits, but instead of using that to call a function we just define the correct parameter as a hashtable.